code/game/machinery/PDApainter.dm

```diff
@@ -144,6 +207,20 @@
 	else
 		to_chat(usr, "<span class='notice'>The [src] is empty.</span>")
 
+/obj/machinery/pdapainter/verb/ejectid()

```

Comment:
These verbs are a crime against UX. Remove them and add an Alt+Click eject. Also put it in the description of the thing, in bold.

`\n<b>Alt-Click</b> to eject its contents!`

Reply:
Also make them one proc, not two.

Reply:
```
/obj/machinery/pdapainter/AltClick(mob/user)
* range check *
* incapacitated check *
* eject PDA *
* eject ID *
* mucho bene *
```